# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1451407527 18000
#      Tue Dec 29 11:45:27 2015 -0500
# Node ID ea779f11deaeb29debc75e3a04be2f99c5cd1377
# Parent  936b88598a0b39d03e49f4da63a920ad62da6ada
strip trailing whitespace from files

* input.cc, oct-obj.h, __init_fltk__.cc: Strip trailing whitespace.

diff --git a/libinterp/corefcn/input.cc b/libinterp/corefcn/input.cc
--- a/libinterp/corefcn/input.cc
+++ b/libinterp/corefcn/input.cc
@@ -1458,17 +1458,17 @@ Undocumented internal function.\n\
 
   octave_value_list retval;
 
   if (nargin == 0)
     retval = ovl (Vgud_mode);
   else
     Vgud_mode = args(0).bool_value ();
 
-  return retval; 
+  return retval;
 }
 
 DEFUN (filemarker, args, nargout,
        "-*- texinfo -*-\n\
 @deftypefn  {} {@var{val} =} filemarker ()\n\
 @deftypefnx {} {@var{old_val} =} filemarker (@var{new_val})\n\
 @deftypefnx {} {} filemarker (@var{new_val}, \"local\")\n\
 Query or set the character used to separate the filename from the subfunction\n\
diff --git a/libinterp/corefcn/oct-obj.h b/libinterp/corefcn/oct-obj.h
--- a/libinterp/corefcn/oct-obj.h
+++ b/libinterp/corefcn/oct-obj.h
@@ -22,9 +22,9 @@ along with Octave; see the file COPYING.
 
 ////////////////////////////////////////////////////////////////////////
 // Note: oct-obj.h was deprecated in version 4.2 and will be removed in
 // version 4.6.
 ////////////////////////////////////////////////////////////////////////
 
 #warning oct-obj.h has been deprecated; use ovl.h instead
 
-#include "ovl.h" 
+#include "ovl.h"
diff --git a/libinterp/dldfcn/__init_fltk__.cc b/libinterp/dldfcn/__init_fltk__.cc
--- a/libinterp/dldfcn/__init_fltk__.cc
+++ b/libinterp/dldfcn/__init_fltk__.cc
@@ -2344,17 +2344,16 @@ Undocumented internal function.  Calls F
 {
 #ifdef HAVE_FLTK
   Fl::check ();
 
   if (Vdrawnow_requested)
     Fdrawnow ();
 
   return ovl ();
-  
 #else
   error ("__fltk_check__: not available without OpenGL and FLTK libraries");
 #endif
 }
 
 // Initialize the fltk graphics toolkit.
 
 DEFUN_DLD (__init_fltk__, , ,
